path/src/update.sh

```
@@ -223,7 +228,30 @@ fount_update_to_ref() {
 }
 
-# After the first successful deno upgrade, routine starts refresh in the background.
+# `fount update <remote-url>` —— 把 origin 指到该远程并切到其默认分支（随后普通更新跟随它）。
+fount_update_to_url() {
+	local url="$1" default_branch symref
+	if invoke_repo_git remote | grep -qx origin; then
+		invoke_repo_git remote set-url origin "$url"
+	else
+		invoke_repo_git remote add origin "$url"
+	fi
+	if [ "$?" -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)